cli/src/index.ts

```
@@ -662,45 +660,45 @@ const pokerInteractiveAction = async (tableAddress: string, address: string) =>
-            case "call":
+            case PlayerActionType.CALL:
                 console.log(chalk.green("Calling..."));
                 await client.playerAction(tableAddress, PlayerActionType.CALL, "", nonce);
                 break;
-            case "deal":
+            case NonPlayerActionType.DEAL:
                 console.log(chalk.green("Deal..."));
 
                 // use native crypto functions to generate a seed
                 const seed = crypto.randomBytes(32).toString("hex");
                 const publicKey = getPublicKeyDetails(pk);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)